lz4-compress/decompress.rs

```diff
@@ -0,0 +1,8 @@
+#![no_main]
+
+#[macro_use] extern crate libfuzzer_sys;
+extern crate lz4_compress;
+
+fuzz_target!(|data: &[u8]| {
+    if let Ok(_) = lz4_compress::decompress(data) {}
```

Comment:
The concern this there (https://github.com/rust-fuzz/targets/pull/40#issuecomment-285976312) but I only did it this way to easily add a `print!(".")` for debugging :)